test_psa_target.py

```diff
@@ -173,6 +173,10 @@ def _erase_flash_storage(args, suite):
             "0xA000000",
             "-fill",
             "0xFF",
+            "0x0200000",
+            "0x0210000",
+            "-fill",
+            "0xFF",
```

Comment:
This is according to https://git.trustedfirmware.org/TF-M/trusted-firmware-m.git/tree/platform/ext/target/musca_s1/partition/flash_layout.h?h=TF-Mv1.3.0#n52 and the size is aligned to 0x10000 to match Musca S1's DAPLink's requirement.